code_schemes/dadaab_s01e01_reasons.json

```diff
@@ -70,9 +70,17 @@
     {
       "CodeID": "code-f638dc8b",
       "CodeType": "Normal",
-      "DisplayText": "negative culture",
+      "DisplayText": "culture or beliefs",
       "NumericValue": 9,
-      "StringValue": "negative_culture",
+      "StringValue": "culture or beliefs",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-26bbc018",
+      "CodeType": "Normal",
+      "DisplayText": "religion",
+      "NumericValue": 11,
+      "StringValue": "religion",
```

Comment:
Coding frame also has "fear of shame & embarrassment" now

Reply:
That remains unchanged in numericvalue 8